components/builder-api/habitat/plan.sh

```diff
@@ -9,7 +9,10 @@ pkg_deps=(core/glibc core/openssl core/coreutils core/gcc-libs core/zeromq core/
   core/libarchive)
 pkg_build_deps=(core/protobuf core/protobuf-rust core/coreutils core/cacerts
   core/rust core/gcc core/git core/pkg-config core/node core/phantomjs)
-pkg_expose=(9636)
```

Comment:
@fnichol it's always been called pkg_expose. Maybe I should rename it now since this is probably the right time to break things anyway ;)